source/quickstart/windows.rst

```diff
@@ -6,31 +6,115 @@ Quickstart for Windows
 
 .. default-domain:: minio
 
-This page deploys MinIO on Windows for use with early development
-and evaluation environments.
+.. |OS| replace:: Windows
 
-For instructions on deploying to production environments,
-see :ref:`deploy-minio-distributed`.
+Use the procedures on this page to deploy MinIO in :guilabel:`Standalone Mode` on a single local |OS| harddrive as a single MinIO server process (what we call "filesystem mode") with a :ref:`subset of S3 features<minio-installation-comparison>`.
+The MinIO server provides an S3 access layer to the drive or volume with no :ref:`erasure coding <minio-erasure-coding>` or associated features.
+
+Use :guilabel:`Standalone Mode` for early development and evaluation environments.
+For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.
 
 Prerequisites
 -------------
 
-ToDo
+- Read, write, and execute permissions for the user's directory
```

Comment:
On windows I assume it has to be on the preferred local directory and folder path vs a user directory. It could just as easily be at the root of C:/